evil-visual-mark-mode.el

```diff
@@ -109,9 +108,10 @@ This function is called on `evil-normal-state-entry-hook'."
 This function is called on the initialization of
 `evil-visual-mark-mode'"
   (evil-visual-mark-populate-overlay-alist)
-  (--each evil-visual-mark-overlay-alist
-    (evil-visual-mark-overlay-put (car it)
-                                  (cdr it))))
+  (when (evil-normal-state-p)
```

Comment:
If enable evil-visual-mark-mode in evil-insert-state, the marks will be visible immediately before returning to evil-normal-state.
Thanks for the review. The commit message may be confusing, my English is poor, ;-).